### PR TITLE
Update security documentation around  username resolution

### PR DESCRIPTION
**Pull request by scrawfor99**
### Description
This PR clarifies username resolution. 

### Issues Resolved
https://github.com/opensearch-project/documentation-website/issues/4937

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by scrawfor99** [comment]
The last Vale error is just the bot being confused by the parentheses and SSL being an acronym. Everything should be all set.

**Review by kolchfa-aws — APPROVED** [review]
LGTM. Thanks, @scrawfor99!

**Comment by kolchfa-aws** [comment]
@scrawfor99 To what versions is this applicable? We'll need to backport to all of those versions.

File: _security/configuration/tls.md [hunk 1]
```diff
@@ -207,3 +207,24 @@ plugins.security.ssl.http.enabled_protocols:
 ## (Advanced) Disabling client initiated renegotiation for Java 8
 
 Set `-Djdk.tls.rejectClientInitiatedRenegotiation=true` to disable secure client initiated renegotiation, which is enabled by default. This can be set via `OPENSEARCH_JAVA_OPTS` in `config/jvm.options`.
+
+## (Advanced) Using encrypted password settings for SSL   
+
+The default insecure SSL password settings have been deprecated. In order to use the secure alternative of these settings users can use their alternative forms. Specifically, users can append the `_secure` suffix to the SSL settings. The resulting secure alternatives are:
+
```
Comment (vagimeli) [inline _security/configuration/tls.md:214]: Delete extra blank line

**Review by vagimeli — APPROVED** [review]

File: _security/configuration/tls.md (hunk 1 above)
Comment (scrawfor99) [inline _security/configuration/tls.md:214]:
```suggestion
```

Comment (scrawfor99) [reply _security/configuration/tls.md:214]: All set

**Comment by kolchfa-aws** [comment]
@scrawfor99 Yeah, sorry, we'd need to split then

**Comment by scrawfor99** [comment]
@kolchfa-aws should be backported to all then and I will open another.

**Comment by scrawfor99** [comment]
New PR I split into: https://github.com/opensearch-project/documentation-website/pull/5582

**Pull request merged**